server/platform-support/platform-support.rst

```diff
@@ -373,17 +410,35 @@
    * - Windows 10 / Server 2016
      - x86_64
      - Community
+     -
+     - |checkmark|
      - |checkmark|
      - |checkmark|
      - |checkmark|
+
```

Comment:
We are missing macOS 13 support for 7.0. Rows should be added for Enterprise and Community.

Reply:
Fixed.